cmd/notary/tuf.go

```diff
@@ -398,7 +451,8 @@ func verify(cmd *cobra.Command, args []string) {
 		logrus.Error("notary: data not present in the trusted collection.")
 		os.Exit(1)
 	} else {
-		_, _ = os.Stdout.Write(payload)
+		_, _ = output.Write(payload)
+		output.Close()
```

Comment:
`defer output.Close()` is more idiomatic and ensures it will always execute, even if the write panics.